mrjones2014/yaclt
# #61 - fix default changelog template

## What Changed
* default template now has a line break at the end just before the next heading
*   Special characters are no longer HTML encoded when converting to markdown using Handlebars

## How to test
1. Pull this branch
2. build yactl
3. generate a new changelog with special characters and see the results

## Notes
None.

Resolves: #61
